src/components/Text/Text.module.css

```diff
@@ -6,7 +6,7 @@
   text-overflow: ellipsis;
   direction: rtl;
   max-width: 100%;
-  line-height: var(--line-height-2, 20px);
+  /* line-height: var(--line-height-2, 20px); */
```

Comment:
do we need to keep this line for any specific reason?

Reply:
yes I might add it back in or make it it's own class depending on how it looks with the command preview :)